implementations/micrometer-registry-influx/src/main/java/io/micrometer/influx/InfluxMeterRegistry.java

```diff
@@ -22,6 +22,8 @@
 import io.micrometer.core.instrument.util.DoubleFormat;
 import io.micrometer.core.instrument.util.MeterPartition;
 import io.micrometer.core.lang.Nullable;
+
+import org.apache.commons.lang.StringUtils;
```

Comment:
Won't be able to use this because:

```
commons-lang:commons-lang:2.6
\--- commons-configuration:commons-configuration:1.8
     \--- com.netflix.archaius:archaius-core:0.4.1
          \--- com.netflix.hystrix:hystrix-core:1.5.12
               \--- project :micrometer-core
                    \--- compileClasspath
```